dbms/src/Storages/KVStore/MultiRaft/RegionCFDataBase.h

```diff
@@ -25,7 +25,36 @@ namespace DB
 
 struct TiKVRangeKey;
 using RegionRange = RegionRangeKeys::RegionRange;
-using RegionDataRes = int64_t;
+struct RegionDataMemDiff
+{
+    using Type = Int64;
+    Type payload;
+    Type decoded;
+
+    template <typename T1, typename T2>
+    RegionDataMemDiff(T1 payload_, T2 decoded_)
+        : payload(payload_)
+        , decoded(decoded_)
+    {}
+    RegionDataMemDiff()
+        : payload(0)
+        , decoded(0)
+    {}
+
+    RegionDataMemDiff operator-() const { return {-payload, -decoded}; }
```

Comment:
I'd like to keep a function naming `negative` but not overload the `operator-`. Because the other functions in `RegionDataMemDiff` use `add` `sub` but not overload the related operators
```suggestion
    RegionDataMemDiff negative() const { return {-payload, -decoded}; }
```

Reply:
fixed